### DISCUSSION TITLE
实操 合约 create 合约, Pair 合约 输出 token0 和 token1 都是 黑洞地址?

### DISCUSSION THREAD

**Discussion by TayueV** [AmazingAng/WTF-Solidity#550] [Q&A] [answered]
大佬们   用 create , 实现 极简版的 uniswap ,  调用 工厂合约的 createPair 方法 ,传入两个地址参数,
然后再 调用 pair 合约, 输出的 token0 和 token1 都是 0x000000000 , 为什么啊?

**Comment by steven4242** [comment]
在部署的下面有一个输入 At Address的地方，把你得到的Pair地址输进去，在去使用函数就可以了

**Comment by steven4242** [comment]
![Uploading 图像2024-1-30 16.39.jpeg…]()

**Comment by XdpCs** [comment, marked as answer]
请详细描述问题，我今天重新测试这个代码，输出结果是正确的，如有需要，可以重新开启这个讨论
